mslib/utils/mssautoplot.py

```diff
@@ -289,7 +289,7 @@ def draw(self):
 
                 xmls = wms.getmap(**kwargs)
 
-                if not type(xmls) == 'list':
+                if not isinstance(xmls, list):
```

Comment:
likly this got merged in?

Reply:
Since we enabled flake8, I got an error when running test cases because of that, I was updated

Reply:
Ok. That explains it. Keep it.
It was also some time ago fixed in stable/develop.